Description: Fixes path to nec-plugin.filters
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-03-26

--- networking-nec-2015.1~b2+git20150326+710a80087d.orig/setup.cfg
+++ networking-nec-2015.1~b2+git20150326+710a80087d/setup.cfg
@@ -26,7 +26,7 @@ packages =
     networking_nec
 data_files =
     etc/neutron/rootwrap.d =
-        etc/rootwrap.d/nec-plugin.filters
+        etc/neutron/rootwrap.d/nec-plugin.filters
     etc/neutron/plugins/nec = etc/neutron/plugins/nec/nec.ini
 
 [global]
