Description: Removes base.override_nvalues() calls
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-03-26

--- networking-nec-2015.1~b2+git20150326+710a80087d.orig/networking_nec/tests/unit/openflow/test_l2_agent.py
+++ networking-nec-2015.1~b2+git20150326+710a80087d/networking_nec/tests/unit/openflow/test_l2_agent.py
@@ -42,7 +42,6 @@ class TestNecAgentBase(base.BaseTestCase
         # TODO(amotoki): It seems a bug of neutorn.context
         # why does get_admin_context_without_session on agent side
         # require policy.json?
-        test_base.override_nvalues()
         super(TestNecAgentBase, self).setUp()
         nec_config.register_agent_opts()
         cfg.CONF.set_default('firewall_driver',
--- networking-nec-2015.1~b2+git20150326+710a80087d.orig/networking_nec/tests/unit/openflow/test_ofc_manager.py
+++ networking-nec-2015.1~b2+git20150326+710a80087d/networking_nec/tests/unit/openflow/test_ofc_manager.py
@@ -40,7 +40,6 @@ class OFCManagerTestBase(base.SqlTestCas
     """Class conisting of OFCManager unit tests."""
 
     def setUp(self):
-        base.override_nvalues()
         super(OFCManagerTestBase, self).setUp()
         driver = ("networking_nec.tests.unit.openflow."
                   "stub_ofc_driver.StubOFCDriver")
--- networking-nec-2015.1~b2+git20150326+710a80087d.orig/networking_nec/tests/unit/openflow/test_plugin.py
+++ networking-nec-2015.1~b2+git20150326+710a80087d/networking_nec/tests/unit/openflow/test_plugin.py
@@ -71,7 +71,6 @@ class NecPluginV2TestCaseBase(object):
         self.ofc_manager_p.start()
 
     def setup_nec_plugin_base(self):
-        base.override_nvalues()
         self._set_nec_ini()
         self.patch_remote_calls()
 
--- networking-nec-2015.1~b2+git20150326+710a80087d.orig/networking_nec/tests/unit/openflow/test_router.py
+++ networking-nec-2015.1~b2+git20150326+710a80087d/networking_nec/tests/unit/openflow/test_router.py
@@ -28,7 +28,6 @@ class NecRouterL3AgentTestCase(test_ext_
     _plugin_name = test_plugin.PLUGIN_NAME
 
     def setUp(self):
-        base.override_nvalues()
         mock.patch(test_plugin.OFC_MANAGER).start()
         super(NecRouterL3AgentTestCase, self).setUp(self._plugin_name)
 
--- networking-nec-2015.1~b2+git20150326+710a80087d.orig/networking_nec/tests/unit/openflow/test_security_group.py
+++ networking-nec-2015.1~b2+git20150326+710a80087d/networking_nec/tests/unit/openflow/test_security_group.py
@@ -35,7 +35,6 @@ NOTIFIER = 'networking_nec.plugins.openf
 class NecSecurityGroupsTestCase(test_sg.SecurityGroupDBTestCase):
 
     def setUp(self, plugin=None):
-        base.override_nvalues()
         test_sg_rpc.set_firewall_driver(test_sg_rpc.FIREWALL_HYBRID_DRIVER)
         mock.patch(NOTIFIER).start()
         mock.patch(OFC_MANAGER).start()
